sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_operations.py

```diff
@@ -262,25 +260,21 @@ def build_event_grid_renew_cloud_event_locks_request(  # pylint: disable=name-to
     return HttpRequest(method="POST", url=_url, params=_params, headers=_headers, **kwargs)
 
 
-class EventGridClientOperationsMixin(EventGridClientMixinABC):
+class EventGridPublisherClientOperationsMixin(EventGridPublisherClientMixinABC):
 
     @distributed_trace
-    def _publish_cloud_event(  # pylint: disable=protected-access
+    def _send(  # pylint: disable=protected-access
         self, topic_name: str, event: _models._models.CloudEvent, **kwargs: Any
     ) -> _models._models.PublishResult:
         # pylint: disable=line-too-long
-        """Publish Single Cloud Event to namespace topic. In case of success, the server responds with an
-        HTTP 200 status code with an empty JSON object in response. Otherwise, the server can return
-        various error codes. For example, 401: which indicates authorization failure, 403: which
-        indicates quota exceeded or message is too large, 410: which indicates that specific topic is
-        not found, 400: for bad request, and 500: for internal server error.
+        """Publish a single Cloud Event to a namespace topic.
 
         :param topic_name: Topic Name. Required.
         :type topic_name: str
         :param event: Single Cloud Event being published. Required.
-        :type event: ~azure.eventgrid.models.CloudEvent
+        :type event: ~azure.eventgrid.models._models.CloudEvent
```

Comment:
is this the right namespace `models._models` ?

Reply:
this is gen code - CloudEvent is a private model so its under ._models